src/components/FormAddUser.vue

```diff
@@ -64,7 +70,46 @@
         this.$store.dispatch('changePhone', {
           text: e.target.value
         })
+      },
+      addUser: function () {
+        if (this.inputUsername === '' || this.inputUsername === null) {
+          alert('Empty username!')
+        } else {
+          let id = this.listUsers.length
+          this.$store.dispatch('addUser', {
+            id,
+            name: this.inputUsername,
+            address: this.inputAddress,
+            phone: this.inputPhone,
+            deleted: false
```

Comment:
Should add comma at the end of object attribute. See javascript convention for more details